app/services/folio_client.rb

```diff
@@ -109,7 +109,6 @@ def get_json(path, **kwargs)
   # @raises [StandardError] if the response was not a 200
   # @return [Hash] the parsed JSON data structure
   def parse_json(response)
-    raise response unless response.status == 200
```

Comment:
This intentionally guarded against a 500 error status, which ended up showing up as a JSON parse error.  I think we should add this back and maybe look for `response.success?` instead.

Reply:
Maybe... although when I tried that, ruby complained about trying to raise a Faraday response object instead of an exception 🤷‍♂️ It seemed like the `check_response` call is already checking the status, so maybe we don't need it?

Reply:
Okay, I may have written this guard prior to adding check_response, so it may have been redundant.